src/scenes/game/Game.tsx

```diff
@@ -93,7 +98,11 @@ export const Game = ({ sharedState, publish }: GameProps) => {
       rebuttal: sharedState.rebuttal ? sharedState.rebuttal : DEFAULT_REBUTTAL,
     };
     publish(action);
-    publish({ type: ActionTypes.START_TURN });
+    if (getGuessScore(sharedState.game) === 4 && isCatchUp(sharedState.game)) {
```

Comment:
has `sharedState` already been updated with the latest score when this line is called? This might cause a bug if not, but we can fix it later and add some tests

Reply:
Seems to be. Just tested again, manually. Let's add tests too!